doc/panes/editor.rst

```diff
@@ -29,32 +31,275 @@ The Editor pane consists of the following areas:
+Providing a description to the right of the separator will give that cell its own name in the :doc:`outline`.
 You can also create "subsections" by adding more ``%`` signs to the cell separator, e.g. ``# %%%`` to create a level 2 subsection, ``# %%%%`` for level 3, etc.
-This displays multiple levels in the the :guilabel:`Outline Explorer`.
+This displays multiple levels in the :doc:`outline`.
 
 .. image:: /images/editor/editor-subsections.png
    :alt: Spyder outline panel, showing an example of sub sections
 
-Note that this only affects how the outline is displayed; it has no impact on the functioning of the code cells.
+.. note:: This only affects how the cell is displayed in the :doc:`outline`, and doesn't affect running it in the Editor.
+
+To run the code in a cell, use :menuselection:`Run --> Run cell`, the :guilabel:`Run cell` button in the toolbar or the keyboard shortcut (:kbd:`Ctrl-Enter`/:kbd:`Cmd-Return` by default).
+You can also run a cell and then jump to the next one, letting you quickly step through multiple cells, using :menuselection:`Run --> Run cell and advance` (:kbd:`Shift-Enter` by default).
+
+
+Automatic formatting
+~~~~~~~~~~~~~~~~~~~~
+
+The Editor has built-in support for automatically formatting your code using several popular tools, including `Autopep8 <https://github.com/hhatto/autopep8>`_, `Yapf <https://github.com/google/yapf>`_, and `Black <https://black.readthedocs.io/en/stable/>`_.
+The :guilabel:`Format file or selection with {tool}` command in the :guilabel:`Source` or context menu will format either the selected fragment (if text is selected) or the entire active file.
+
+.. image:: /images/editor/editor-automatic-formatting.gif
+   :alt: Spyder editor panel, showing an example of code selection formatting
+
+You can have the Editor automatically autoformat a file every time you save your work.
+To set this up, go to :menuselection:`Preferences --> Completion and linting --> Code style and formatting --> Code formatting` and check the :guilabel:`Autoformat files on save` option.
+
+.. image:: /images/editor/editor-autoformat-setting.png
+   :alt: Spyder's preferences dialog, showing checking the autoformat files on save setting
+
+
+
+============
+Running code
+============
+
+The Editor lets you run an entire file as well as specific lines, selections or cells.
+
+As your code is running,
+
+* The :doc:`ipythonconsole` will display output and errors.
+* The :doc:`variableexplorer` allows you to browse and interact with the objects generated.
+* The :doc:`plots` pane renders the figures and images created.
+* The :doc:`profiler` helps you optimize your code's run time.
+
+
+Run file
+~~~~~~~~
+
+Run an entire Editor file using the :menuselection:`Run --> Run` menu item, the :guilabel:`Run file` toolbar button or the :kbd:`F5` key.
+Use :guilabel:`Run --> Re-Run last script` to re-run the most recent file executed with the above.
```

Comment:
Should this be `:menuselection:` instead of `:guilabel:` maybe? Checking the preview this is looking like this:
![imagen](https://user-images.githubusercontent.com/16781833/201737322-82eab743-b0c9-44db-b349-7ee486d0dfd5.png)

Reply:
What do you mean by "this"? I updated the the "Run --> Re-Run last script" if this is what you refer to @dalthviz Thanks

Reply:
Yep, that's exactly right @hyounes4560 :+1: . That was probably my mistake and I missed it, sorry.